libgalois/include/katana/DynamicBitset.h

```diff
@@ -309,12 +309,21 @@ class KATANA_EXPORT DynamicBitset {
   void bitwise_xor(const DynamicBitset& other1, const DynamicBitset& other2);
 
   /**
-   * Count how many bits are set in the bitset
+   * Count how many bits are set in the bitset. Do not call in a parallel
+   * region as it uses a parallel loop.
    *
    * @returns number of set bits in the bitset
    */
   size_t count() const;
 
+  /***
+   * Count number of set bits in the bitset serially. Useful if
+   * you need to count different bitsets on different threads.
+   *
+   * @returns number of set bits in the bitset
+   */
+  size_t SerialCount() const;
```

Comment:
nit: I'd prefer a better API but I don't have concrete suggestions now, so this is fine for now.